Composer/packages/client/src/pages/publish/BotStatusList.tsx

```diff
@@ -48,6 +49,25 @@ export const BotStatusList: React.FC<BotStatusListProps> = ({
 }) => {
   const [expandedBotIds, setExpandedBotIds] = useState<Record<string, boolean>>({});
   const [currentSort, setSort] = useState({ key: 'Bot', descending: true });
+  const [clipboardText, setClipboardText] = useState('');
+  const clipboardTextFieldRef = useRef<ITextField>(null);
+
+  const copyStringToClipboard = (value?: string) => {
+    try {
+      if (clipboardTextFieldRef.current) {
+        setClipboardText(value || '');
+        setTimeout(() => {
```

Comment:
nit: if you create the input field dynamically you dont need setTimeout

Reply:
true